test/e2e/multi-cluster-deployment/README.md

```diff
@@ -37,7 +37,8 @@ $ kubectl config rename-context <default-wec1-context-name> kscore
 2. Run e2e test in your ocp cluster:
 
 ```
- export KUBESTELLAR_VERSION=0.21.2
+ export KUBESTELLAR_VERSION=0.22.0-rc2
  export OCM_STATUS_ADDON_VERSION=0.2.0-rc8
+ export OCM_TRANSPORT_PLUGIN=0.1.5
```

Comment:
This also needs to be added to the release process doc. That would be reasonable to include in this PR, since this PR is introducing this reference. Could instead be done in a follow-on PR.

Reply:
It seems that you have address this in this https://github.com/kubestellar/kubestellar/pull/2101. Let me know if otherwise.

Reply:
I did not (yet). But this does not have to be done in this PR.